src/test/java/com/wikia/webdriver/common/core/CommonUtils.java

```diff
@@ -100,13 +101,13 @@ public static String sendPost(String apiUrl, String[][] param) {
       HttpEntity entity = response.getEntity();
       return EntityUtils.toString(entity);
     } catch (UnsupportedEncodingException e) {
-      PageObjectLogging.log("sendPost", e, false);
+      LOG.error("sendPost", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "sendPost" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)